docs/docs-content/clusters/data-center/nutanix/install-pcg.md

```diff
@@ -10,160 +10,206 @@ tags: ["data center", "nutanix"]
+  ```bash hideClipBoard
+  namespace/mytestnamespace created
+  configmap/user-ca-bundle created
+  secret/mytestcluster created
+  kubeadmconfigtemplate.bootstrap.cluster.x-k8s.io/mytestcluster-kcfg-0 created
+  cluster.cluster.x-k8s.io/mytestcluster created
+  machinedeployment.cluster.x-k8s.io/mytestcluster-wmd created
+  machinehealthcheck.cluster.x-k8s.io/mytestcluster-mhc created
+  kubeadmcontrolplane.controlplane.cluster.x-k8s.io/mytestcluster-kcp created
+  nutanixcluster.infrastructure.cluster.x-k8s.io/mytestcluster created
+  nutanixmachinetemplate.infrastructure.cluster.x-k8s.io/mytestcluster-mt-0 created
+  ```
 
-4. Select **Self Hosted** in the next window that Palette displays.
 
-5. Provide a name for the PCG and use the **drop-down Menu** to select Nutanix as the cloud type. Click **Confirm** to continue. Palette displays the Private Cloud Gateway Overview page. 
+### Install CNI on Workload Cluster
 
-6. To install the Palette agent, copy the kubectl commands from the slide-out panel and execute them against your Kubernetes cluster.
+6. After your Nutanix workload cluster is deployed, retrieve its kubeconfig file with the command described below.
 
-7. Close the slide-out panel when you have copied both the commands. The **Cluster Status** field on the PCG Overview page displays **Pending** while the PCG is deploying. Deployment is complete when the **Cluster Status** field displays **Running**.  
+  ```bash
+  clusterctl get kubeconfig $TEST_CLUSTER_NAME > $TEST_CLUSTER_NAME.kubeconfig -namespace $TEST_NAMESPACE
+  ```
 
 
-## Validate
+7. Deploy a Container Network Interface (CNI) pod in the workload cluster to enable pod-to-pod communication. For more information, refer to [Deploy a CNI solution](https://cluster-api.sigs.k8s.io/user/quick-start.html#deploy-a-cni-solution) in the [Nutanix Quick Start](https://cluster-api.sigs.k8s.io/user/quick-start.htm) reference. [Calico](https://docs.tigera.io/calico/latest/about/) is used as the CNI solution in this example.
 
-When deployed, the PCG registers itself with Palette. Use the steps below to verify if the PCG is registered.
+  ```bash
+  kubectl apply --filename https://raw.githubusercontent.com/projectcalico/calico/v3.26.1/manifests/calico.yaml
+  ```
 
-1. Log in to [Palette](https://console.spectrocloud.com/).
+8. To verify that the CNI was deployed successfully, issue the following command. 
 
+  ```bash
+  kubectl --kubeconfig=./$TEST_CLUSTER_NAME.kubeconfig get nodes
+  ```
 
-2. Navigate to the **left Main Menu** and select **Tenant Settings**.
+  The output should display your nodes with a **Ready** status.
 
+    ```bash hideClipBoard
+    NAME                           STATUS   ROLES           AGE   VERSION
+    test-cluster-kcp-qhb5h         Ready    control-plane   26h   v1.26.7
+    test-cluster-wmd-gdjps-gx267   Ready    <none>          26h   v1.26.7
+    ```
 
-3. Next, on the **Tenant Settings Menu**, select **Private Cloud Gateways**.
+## Validate
```

Comment:
We need to provide guidance about what to do with the kind cluster we stood up. I'm pretty sure we can terminate it once the workload cluster is deployed. @cloudmaniac correct me if I am wrong please.

Reply:
Yes, we can. This was addressed in step 4 of the **Install PCG Validate** section. :)

Reply:
:white_check_mark: